### ISSUE TITLE
GET works but POST, PUT, DELETE show CORS error

### ISSUE THREAD

**Issue by norricorp** [appcela/grails3-cors-interceptor#5]
Hi,
so I have used the plugin and other than change build.gradle, have made no other config changes. I am using grails 3.2.0.M2. I have used the default controller from using the rest profile so essentially all of the code is in the UrlMappings.groovy file.
Before adding the plugin, I was getting CORS error on GET. Since using the plugin, GET now retrieves code. But POST, PUT and DELETE actions still show CORS error in the browser console. (I am using an angular app in nginx getting data from the grails app running under tomcat). 
Do I need to make other changes to the grails code?
Regards,
John

**Comment by ichangxappcela** [comment]
Can you try grails 3.2.0?

**Comment by norricorp** [comment]
I am currently on 3.2.1 as that has built in CORS support but that
apparently does not work either so have been told to move to 3.2.2 when
that is released. 

On 2016-10-28 6:39, appcela wrote: 

> Can you try
> grails 3.2.0? 
> 
> ## 
> 
> You are receiving this because you authored the
> thread.
> Reply to this email directly, view it on GitHub [1], or mute
> the thread [2].

## Links:

[1]
https://github.com/appcela/grails3-cors-interceptor/issues/5#issuecomment-256840622
[2]

**Comment by ichangxappcela** [comment]
Yeah, the new CORS support introduced in Grails 3.2.1 doesn't really work. There are other issues with Grails 3.2.1 as well. 

In any case, I have a fix for Grails 3.2.1 now.

Please use v1.2.1,

```
compile "org.grails.plugins:grails3-cors-interceptor:1.2.1"
```

Enjoy!

**Comment by norricorp** [comment]
This seems to have done the trick - many thanks.

------ Original Message ------
